### PR TITLE
New Adapter: Aceex

### PR DESCRIPTION
**Pull request by supportAceex**

### CODE REVIEW

File: static/bidder-info/aceex.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+maintainer:
+  email: "tech@aceex.io"
+gvlVendorID: 0
```
Comment (SyntaxNode) [inline static/bidder-info/aceex.yaml:3]: If you are not registered with IAB Europe for GDPR, please remove this line entirely instead of setting it to 0.

Comment (supportAceex) [reply static/bidder-info/aceex.yaml]: @SyntaxNode Thank you for your review! Fixed

File: adapters/aceex/aceex.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+		bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
+			Bid:     &bid,
+			BidType: getMediaTypeForImp(bid.ImpID, openRTBRequest.Imp),
+		})
+	}
+	return bidResponse, nil
+}
+
+func getMediaTypeForImp(impId string, imps []openrtb2.Imp) openrtb_ext.BidType {
+	mediaType := openrtb_ext.BidTypeBanner
```
Comment (guscarreon) [inline adapters/aceex/aceex.go:182]: The current logic makes `getMediaTypeForImp` to return `openrtb_ext.BidTypeBanner` even if we didn't find a matching `imp.ID` in line 184. Other adapters throw an error if the `imp.ID` didn't find a match. Does that make sense in the context of your adapter? This is not a requirement, I'm ok either way.

**Review by guscarreon — COMMENTED** [review]
Code looks pretty solid and very well covered by unit tests. Left a couple of questions

File: adapters/aceex/aceex.go [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type AceexAdapter struct {
```
[the remark this answers is not in the document]
Comment (supportAceex) [reply adapters/aceex/aceex.go]: fixed!

File: adapters/aceex/aceex.go [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+}
+
+func (a *AceexAdapter) buildEndpointURL(params *openrtb_ext.ExtAceex) (string, error) {
+	endpointParams := macros.EndpointTemplateParams{AccountID: params.AccountID}
+	return macros.ResolveMacros(a.endpoint, endpointParams)
+}
+
+func (a *AceexAdapter) checkResponseStatusCodes(response *adapters.ResponseData) error {
+	if response.StatusCode == http.StatusNoContent {
+		return nil
```
[the remark this answers is not in the document]
Comment (supportAceex) [reply adapters/aceex/aceex.go:114]: fixed!

**Review by guscarreon — COMMENTED** [review]
@supportAceex thank you for adressing our feedback. Your code looks pretty good but test case `adapters/aceex/aceextest/supplemental/status-code-no-content.json` began failing in this last commit. Can you please correct before we approve?

**Review by AlexBVolcy — COMMENTED** [review]
I also think the code looks solid overall, but like Gus mentioned, there's a test that needs to be fixed before approval

**Comment by supportAceex** [comment]
Thank you! Fixed!

**Review by guscarreon — APPROVED** [review]
LGTM

**Review by hhhjort — APPROVED** [review]
LGTM

**Pull request merged**